config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -13,6 +13,6 @@ periodics:
     path_alias: "sigs.k8s.io/cluster-api-provider-aws"
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190214-f4092ae69-1.12
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190219-caa48e6c3-1.12
```

Comment:
rotation missing.

Reply:
Pattern of rotation missing very common in this PR.

Reply:
no we don't care about cluster api

Reply:
will let repo owner figure out when their repo want to move forward, instead of flip version and break their code base